lib/src/geohash_base.dart

```diff
@@ -162,8 +163,8 @@ class Geohash {
     if (identical(1.0, 1)) {
       // Some of our intermediate numbers are STILL too big for javascript,
       // so  we use floating point math...
-      final longitudeDiff = pow(2.0, 52 - longitudeBits);
-      final latitudeDiff = pow(2.0, 52 - latitudeBits);
+      final longitudeDiff = pow(2.0, 52 - longitudeBits).toDouble();
```

Comment:
I know I'm personally terrible about comments, but could you please put comment as to why this is required?

Reply:
is it just a type checking issue?